features/user.feature

```diff
@@ -424,6 +424,17 @@ Feature: Manage WordPress users
       testuser4@example.com
       """
 
+  Scenario: Invalid User Role
+    Given a WP install
+
+    When I run `wp user update admin --role=banana`
+    Then STDOUT should contain:
+      """
+      Warning:
+      """
+
```

Comment:
Can we add an assertion for the admin's role after this is run?

Reply:
Can you help me with the functional testing I haven't written any before I am writing this for the first time.

Reply:
This can be done be adding
```
When I run `wp user get testuser4@example.com --field=roles`
Then STDOUT should be:
  """
  administrator
  """
```

Reply:
@wojsmol It should be admin instead of the testuser4@example.com right?